rule/expr.go

```diff
@@ -382,6 +422,45 @@ func (v *Value) Equal(other *Value) bool {
 	return v.compare(token.EQL, other)
 }
 
+// GT reports whether v is greater than other.
+func (v *Value) GT(other *Value) bool {
+	switch v.Type {
+	case "bool":
+		v1, _ := strconv.ParseBool(v.Data)
```

Comment:
Why ignoring the error when you can return it?

Reply:
Because the function won't fail, I am sure that I will parse the correct type.
It's impossible to have a value in `v.Data` that mismatches the type in `v.Type`.

Reply:
What if I create it by hand in the seeds scripts for example?

Reply:
The only way to create `Value`s is to use the `rule.TValue` functions (where `T` is a type) and those are strongly typed thus I don't see how we can have a value that differs from the type?

Reply:
I'm just being picky here, but Regula is a lib and you don't know how people might want to use it:

```go
v := rule.Value{
  Type: "int64",
  Name: "foo",
  Data: fmt.Sprintf("%f", math.Pi / 2), // oups it's a float
}
```

Reply:
From that perspective, you are right. But for this "intermediate" version I think that we can ship it like that and be more carefully in the next one.

Reply:
I don't think intermediate versions allow us to be less careful about what we are producing. I agree that we should not make too much effort for nothing, but here we are talking about adding an `if err != nil` that doesn't cost much.

Reply:
> Regula is a lib and you don't know how people might want to use it

This argument is enough to add the check. From time to time I forgot that the users can do whatever they want with the exported type and my opinion is based on what they are doing and how they are using it at the moment.
Will add the check in #107.